src/opera/commands/undeploy.py

```diff
@@ -23,7 +23,7 @@ def undeploy(args):
     if args.instance_path and not path.isdir(args.instance_path):
         raise argparse.ArgumentTypeError("Directory {0} is not a valid path!".format(args.instance_path))
 
-    storage = Storage(Path(args.instance_path).joinpath(".opera")) if args.instance_path else Storage(Path(".opera"))
+    storage = Storage(Path(args.instance_path) / ".opera") if args.instance_path else Storage(Path(".opera"))
```

Comment:
```suggestion
    storage = Storage(Path(args.instance_path)) if args.instance_path else Storage(Path(".opera"))
```
As above.

Reply:
This was also modified as you suggested.